<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovCom</class>
 <widget class="QWidget" name="MovCom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT FILES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_update">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE LIST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_quit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="list_window">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>411</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_com">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>MAKE COMPARISIONS</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECTED FILES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>FETCHED FILES:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_fetched">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>321</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_quit</sender>
   <signal>clicked()</signal>
   <receiver>MovCom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
